pkg/postgres/metrics.go

```diff
@@ -1,30 +1,28 @@
 package postgres
 
 import (
-	"strings"
 	"time"
 
 	"github.com/jackc/pgx/v4"
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/stackrox/rox/pkg/metrics"
-	"github.com/stackrox/rox/pkg/stringutils"
 )
 
 func init() {
 	prometheus.MustRegister(
-		queryDuration,
+		//queryDuration,
 		queryErrors,
 	)
 }
 
 var (
-	queryDuration = prometheus.NewHistogramVec(prometheus.HistogramOpts{
-		Namespace: metrics.PrometheusNamespace,
-		Subsystem: metrics.CentralSubsystem.String(),
-		Name:      "postgres_query_duration",
-		Help:      "Time in ms for a query to execute",
-		Buckets:   prometheus.ExponentialBuckets(4, 2, 16),
-	}, []string{"scope", "query"})
+	//queryDuration = prometheus.NewHistogramVec(prometheus.HistogramOpts{
+	//	Namespace: metrics.PrometheusNamespace,
+	//	Subsystem: metrics.CentralSubsystem.String(),
+	//	Name:      "postgres_query_duration",
+	//	Help:      "Time in ms for a query to execute",
+	//	Buckets:   prometheus.ExponentialBuckets(4, 2, 16),
+	//}, []string{"scope", "query"})
```

Comment:
Why not removing? When this will be uncommented?

Reply:
I can just remove it for now. In general, I do want to have a way to get a way to track the timing of some of the bespoke queries that we are running